src/index.html

```diff
@@ -236,5 +298,18 @@ <h1 class="text-white">How can we help?</h1>
         </div>
       </div>
     </section>
+    <script>
```

Comment:
If you move the tag with `index.js` down here to the bottom then you won't need to have this separate inline `script` tag and you can put everything in `index.js`. That said, for code-organization purposes, you may want to divide functionality out by section and have multiple `.js` files down here, maybe something called `hero.js` which could contain the carousel (and eventually typewriter) code, and then a separate file for the other sections of the page.

Reply:
Thanks for pointing this out--for some reason I assumed the `index.js` tag had to go in the header, so that's good to know. That being said, I'm inclined to leave it for now because I do think I'll be tempted to use separate `.js` files; I can move this down later if I change my mind.